### PR TITLE
Non interactive setup

### PR DESCRIPTION
**Pull request by yimuniao**

### CODE REVIEW

**Comment by yimuniao** [comment]
Refer to https://github.com/awslabs/collectd-cloudwatch/pull/47
Author claimed that it is under MIT license control.

File: src/setup.py [hunk 1]
```diff
@@ -455,6 +571,37 @@ def _configure_plugin_installation_method(self):
...
+        logger.info('SECRET KEY: {}'.format(self.config.secret_key))
+        logger.info('ACCESS KEY: {}'.format(self.config.access_key))
+        logger.info('HOST: {}'.format(self.config.host))
+        logger.info('PROXY NAME: {}'.format(self.config.proxy_server_name))
+        logger.info('PROXY PORT: {}'.format(self.config.proxy_server_port))
+        logger.info('CREDENTIALS PATH: {}'.format(self.config.credentials_path))
+        logger.info('METHOD ONLY ADD PLUGIN: {}'.format(self.config.only_add_plugin))
+        logger.info('USE RECOMMENDED CONFIG: {}'.format(self.config.use_recommended_collectd_config))
+        logger.info('REGION: {}'.format(self.config.region))
+        logger.info('IAM ROLE (metadata): {}'.format(self.metadata_reader.get_iam_role_name()))
```
Comment (yimuniao) [inline src/setup.py]: Could you add enable_high_resolution_metrics and flush_interval_in_seconds parameters output here?

File: src/setup.py [hunk 2]
```diff
@@ -598,6 +745,121 @@ def main():
...
+    )
+    parser.add_argument(
+        '-p', '--proxy_port', required=False,
+        help='Proxy server port',
+        metavar='PORT', default=None
+    )
+    parser.add_argument(
+        '-e', '--enable_high_resolution_metrics', required=False,
+        help='Enable high resolution metrics',
+        metavar='ENABLE_HIGH_RESOLUTION_METRICS', default=None
```
Comment (yimuniao) [inline src/setup.py]: Could you change the default value to false?

if it is a string, line 486 will always be true.
    line 486: if self.enable_high_resolution_metrics:

File: src/setup.py (hunk 1 above)
Comment (dgouldin) [reply src/setup.py]: Done.

File: src/setup.py (hunk 2 above)
Comment (dgouldin) [reply src/setup.py]: Done.

File: src/setup.py (hunk 1 above)
Comment (yimuniao) [reply src/setup.py]: Sorry, I tested with "./setup.py -I -e ", as "e" is a boolean parameter, it should be the same as push_asg,  could you fix it?

```
[root@ip-172-31-24-132 ~]# ./setup.py -I -e 
usage: a.py [-h] [-I] [-H HOST_NAME] [-r REGION] [-n NAME] [-p PORT]
            [-e ENABLE_HIGH_RESOLUTION_METRICS] [-f FLUSH_INTERVAL]
            [-a ACCESS_KEY] [-s SECRET_KEY] [-P CREDENTIALS_PATH]
            [-m recommended|add|not_modify] [-g] [-c] [-v DIMENSION_VALUE]
            [-d] [-D]

```

**Pull request merged**